### PR TITLE
Revert move of code from uniter to common

### PR DESCRIPTION
**Pull request by wallyworld**
## Description of change

A previous PR extracted some hook command code from uniter and put it in a common package to share between the operator and uniter. However, it didn't work out because the Context objects used with the commands are different and can't easily be consolidated.

The PR reverts that change and also:
- renames jujuc/testing to jujuc/jujuctesting
- deletes some unused code from the uniter

## QA steps

bootstrap and deploy and relate some charms

### CODE REVIEW

File: worker/common/runner/logger.go [hunk 1]
```diff
@@ -12,15 +12,28 @@ import (
 	"github.com/juju/loggo"
 )
 
-type hookLogger struct {
+var logger = loggo.GetLogger("juju.worker.common.runner")
+
+// MewHookLogger creates a new hook logger.
```
Comment (axw) [inline worker/common/runner/logger.go]: s/Mew/New/

File: worker/meterstatus/isolated.go [hunk 2]
```diff
@@ -9,10 +9,10 @@ import (
 	"github.com/juju/errors"
 	"github.com/juju/utils/clock"
 	"gopkg.in/juju/charm.v6/hooks"
-	worker "gopkg.in/juju/worker.v1"
+	"gopkg.in/juju/worker.v1"
 	"gopkg.in/tomb.v1"
 
-	"github.com/juju/juju/worker/uniter/runner/context"
+	"github.com/juju/juju/worker/common/runner"
```
Comment (axw) [inline worker/meterstatus/isolated.go]: "runner" is far too generic without the context of uniter in the path. Maybe "charmrunner"?

File: cmd/juju/commands/debughooks.go [hunk 3]
```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/juju/juju/apiserver/params"
 	"github.com/juju/juju/cmd/modelcmd"
 	"github.com/juju/juju/network/ssh"
-	unitdebug "github.com/juju/juju/worker/uniter/runner/debug"
+	unitdebug "github.com/juju/juju/worker/common/runner/debug"
```
Comment (axw) [inline cmd/juju/commands/debughooks.go]: as discussed, please revert this and we can move it if we implement debug-hooks for CAAS charms

File: worker/meterstatus/isolated.go (hunk 2 above)
Comment (wallyworld) [reply worker/meterstatus/isolated.go]: done

File: worker/common/runner/logger.go (hunk 1 above)
Comment (wallyworld) [reply worker/common/runner/logger.go]: fixed

**Comment by wallyworld** [comment]
$$merge$$

**Comment by wallyworld** [comment]
$$merge$$

**Pull request merged**